bindings/python/pymongoarrow/__init__.py

```diff
@@ -29,10 +30,12 @@ def _parse_version(version):
 
 try:
     from pymongoarrow.lib import libbson_version
-except ImportError as e:
+except ImportError:
     warnings.warn(
-        f"Could not find compiled pymongoarrow.lib extension, please install from source or report the following traceback on the issue tracker:\n{e}"
+        "Could not find compiled pymongoarrow.lib extension, please install "
+        "from source or report the following traceback on the issue tracker"
     )
+    warnings.warn(traceback.print_exc())  # type: ignore[arg-type]
```

Comment:
The arg-type is a real bug. This need to use format_exc. Ideally, the format_exc() would be included in the warning above so the user doesn't see two UserWarnings.

Reply:
ACK